src/Components/Import/Import.module.css

```diff
@@ -0,0 +1,130 @@
+.renderMedia {
+  width: 100%;
+  height: auto; 
+  object-fit: contain;
+}
+.canvaHomePage {
+  background-color: rgb(250, 250, 250);
+  /* background-image: url('../../Media/canvaHomePage.png'); */
+  height: 100vh;
+  width: 100vw;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 0 auto;
+  background-size: cover;
+}
+.fileUploader {
+  width: 300px;
+  height: 150px;
+  border: 2px solid steelblue;
+  border-radius: 8px;
+  padding: 10px;
+  font-size: 16px;
+}
+
+.fileUploaderDragging {
+  background-color: lightblue;
+}
+
+.fileUploaderContents {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.fileUploaderFileName {
+  font-weight: 700;
+}
+
+.dropModal {
+  background-color: rgb(0, 196, 204, 50%);
```

Comment:
Its a good idea to keep colours as constants, that way you can ensure consistency